api/src/main/java/org/apache/iceberg/expressions/InclusiveMetricsEvaluator.java

```diff
@@ -52,7 +52,7 @@ private MetricsEvalVisitor visitor() {
     return visitors.get();
   }
 
-  InclusiveMetricsEvaluator(Schema schema, Expression unbound) {
+  public InclusiveMetricsEvaluator(Schema schema, Expression unbound) {
```

Comment:
TODO: this is temporary until we figure out case sensitivity for metrics evaluators.

Reply:
Let's address this in #413 together with other places.